packages/x-components/src/x-modules/search/components/promoted.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <a :href="item.url" class="x-promoted" data-test="promoted">
+    <img :src="item.image" class="x-promoted__image" alt="" />
```

Comment:
The empty alt is to skip the accesibility for this image. We don't have a field to add a descriptive text to the image, and we are already showing the title, which is the closer we can get to that descriptive text.